web/src/utils/jwt.ts

```diff
@@ -25,6 +25,29 @@ export const generateJWT = async (userId: string): Promise<string> => {
   });
 };
 
+export const generateJWTForEmailVerification = async (
+  userId: string
+): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    sign(
+      { userId },
+      env.JWT_PRIVATE_KEY,
+      { expiresIn: "365d" },
```

Comment:
We might want to make this expire in 20 mins. What do you think @rob-3 @Sleepingpirates ?